src/librbd/librbd.cc

```diff
@@ -7293,6 +7362,43 @@ extern "C" int rbd_group_snap_list_cleanup(rbd_group_snap_info_t *snaps,
   return 0;
 }
 
+extern "C" int rbd_group_snap_list2(rados_ioctx_t group_p,
+                                    const char *group_name,
+                                    rbd_group_snap_info2_t *snaps,
+                                    size_t *snaps_size)
+{
+  // FIPS zeroization audit 20191117: this memset is not security related.
+  memset(snaps, 0, sizeof(*snaps) * *snaps_size);
```

Comment:
This can be moved to line 7386.

Reply:
I think this memset should be removed altogether:

> [...] (and I'm not convinced that memset is needed either -- see the comment on "random junk").